### PR TITLE
Ensure `gpg-agent` is installed on Ubuntu

### PR DESCRIPTION
**Pull request by coaxial**
### Proposed changes

Makes sure `gpg-agent` is present on Ubuntu systems before importing the apt keys, as outlined in #507.

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/ansible-role-nginx/blob/main/CONTRIBUTING.md) document
- [x] I have added Molecule tests that prove my fix is effective or that my feature works
- [x] I have checked that any relevant Molecule tests pass after adding my changes
- [x] I have updated any relevant documentation ([`defaults/main/*.yml`](https://github.com/nginxinc/ansible-role-nginx/blob/main/defaults/main/), [`README.md`](https://github.com/nginxinc/ansible-role-nginx/blob/main/README.md) and [`CHANGELOG.md`](https://github.com/nginxinc/ansible-role-nginx/blob/main/CHANGELOG.md))

### CODE REVIEW

**Comment by alessfg** [comment]
I would probably add the package requirement here instead https://github.com/nginxinc/ansible-role-nginx/blob/main/vars/main.yml#L65 given it seems to be a hard requirement for Debian based systems.

**Comment by coaxial** [comment]
Thanks for the feedback, I have made the change.

Does this require a Molecule test added?

**Comment by alessfg** [comment]
I think it's safe to assume most systems have `gpg-agent` installed. I do not know why the image created by geerlingguys's role does not. I think this time around a test won't be necessary. That being said, can you update the Changelog?

**Comment by coaxial** [comment]
Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.23.2 (April 11, 2022)
```
Comment (alessfg) [inline CHANGELOG.md:3]:
```suggestion
## 0.23.2 (Unreleased)
```

**Pull request merged**